trulens_eval/examples/snowflake_demo/llm.py

```diff
@@ -155,13 +156,24 @@ def generate_response(
                 final_response += chunk
 
         return _reencode_outputs(final_response)
+    
+    @instrument
+    def filter_contexts(self, query: str, contexts: list[str], threshold: float = .5):
```

Comment:
Can we use the native TruLens filter here?

Reply:
Saw your other PR. Let's use the built in guardrailing now that it's checked in. Will close this one